io.github.vinser.pixyne.yml

```diff
@@ -0,0 +1,36 @@
+app-id: io.github.vinser.pixyne
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+    - org.freedesktop.Sdk.Extension.golang
+command: pixyne
+
+finish-args:
+    - --share=ipc 
+    - --socket=x11
+    - --device=dri
+    # We want user fs access so we can read and write folders with jpeg photo files, save user preferences
+    - --filesystem=xdg-config/fyne
```

Comment:
This is not allowed by default. It should use $XDG_CONFIG_HOME to store configs

Reply:
Pixyne app is based on cross-platform [Fyne UI toolkit](https://github.com/fyne-io/fyne) written in Go. On Linux OSes they keep preferences.json in /home/<user>/.config/fyne/io.github.vinser/pixyne directory.
Please advice me how can I set this?
--filesystem=xdg-config-home makes compile error

Reply:
error: Unknown filesystem location xdg-config-home, valid locations are: host, host-os, host-etc, home, xdg-*[/…], ~/dir, /dir

Reply:
How to spell $XDG_CONFIG_HOME in manifest?

Reply:
Where? I'm seeing fyne is correctly using XDG_CONFIG_HOME https://github.com/fyne-io/fyne/blob/a09bbd790b98120bc406a064f80aeafb4ca97656/app/app_xdg.go#L156 

https://pkg.go.dev/os#UserConfigDir

> On Unix systems, it returns $XDG_CONFIG_HOME as specified by https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html if non-empty, else $HOME/.config. 

XDG_CONFIG_HOME is set inside the sandbox, you don't need to do anything

Reply:
Thanks! bbhtt helped me to guess what I have to do 👍🏼